### PR TITLE
Delete contact feature by Dikshya

### PR DESCRIPTION
**Pull request by Dikshyapoudel12**
# Description:
 This is a Gherkin File written for the Delete Contact Feature in Contact Management App.
## Issue:
https://github.com/SagarGi/FeatureFiles/issues/5

### CODE REVIEW

File: Features4/tests/feature/deletContact.feature [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    As user
+    I want to delet added Contact
+    So that I can manage unwanted contact
+
+Background:
+    Given I navigate to the login page
+    When I submit Email and password
+    And  Click on the "Login" button
+    Then I should be logged in 
+    Then Added Contact should be exists
```
Comment (SagarGi) [inline Features4/tests/feature/deletContact.feature]: i have told to use only given step in background

File: Features4/tests/feature/deletContact.feature [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+Background:
+    Given I navigate to the login page
+    When I submit Email and password
+    And  Click on the "Login" button
+    Then I should be logged in 
+    Then Added Contact should be exists 
+
+Scenario: Delete contacts
+    When  the user click "delete" botton 
+    Then  the contact should be deleted
```
Comment (SagarGi) [inline Features4/tests/feature/deletContact.feature]: what if there is no data?

File: Features4/tests/feature/deletContact.feature [hunk 3]
```diff
@@ -0,0 +1,31 @@
+Feature:
+    As user
+    I want to delet added Contact
```
Comment (kiranadh1452) [inline Features4/tests/feature/deletContact.feature]:
```suggestion
    I want to delete added Contact
```

File: Features4/tests/feature/deletContact.feature [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    So that I can remove the unwanted contact from my contact list.
+
+
+Background:
+#  Assuming that the follwing user has been registerd
+	Given the user has logged in with valid email address "deekshyap@gmail.com" and valid password "123456"
+    And the user has been navigated to the home page
```
Comment (kiranadh1452) [inline Features4/tests/feature/deletContact.feature]:
```suggestion
#  Assuming that the follwing user has been registerd
    Given the user has logged in with valid email address "deekshyap@gmail.com" and valid password "123456"
    And the user has been navigated to the home page
```

File: Features4/tests/feature/deletContact.feature [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+    Scenario: Delete specific contact
+        Given the following users has been added
+            | name | email         | phone      | contact types|
+            | xyz  | xyz@gmail.com | 9875688990 | Personal     |
+            | abc  | abc@gmail.com | 9588509560 | Professional |
+        When the user deletes a contact with email "xyz@gmail.com"  
+        Then the contact with email "xyz@gmail.com" should be removed from contact list
+        But  the contact with email "abc@gmail.com" should appear in  the contact list.
```
Comment (kiranadh1452) [inline Features4/tests/feature/deletContact.feature]:
```suggestion
        But  the contact with email "abc@gmail.com" should appear in the contact list.
```

**Review by kiranadh1452 — CHANGES_REQUESTED** [review]

**Review by kiranadh1452 — APPROVED** [review]

**Pull request merged**